docs/variants/hardkernel_odroid_h4/openness_score.md

```diff
@@ -3,7 +3,7 @@
 This page contains the [Dasharo Openness
 Score](../../glossary.md#dasharo-openness-score) for Hardkernel ODROID H4+
 Dasharo releases. The content of the page is generated with [Dasharo Openness
-Score utility](https://github.com/Dasharo/Openness-Score).
+Score utility v0.2.0](https://github.com/Dasharo/Openness-Score).
```

Comment:
I thought it will be added in each subsection not in the file header. E.g.

```
Report has been generated with Openness Score utility version v0.2

Openness Score for hardkernel_odroid_h4_v0.9.0.rom
```

That's how the latest utility version is supposed to work.

Reply:
https://github.com/Dasharo/docs/compare/3f18ceda588ee4399826d440e08eda8e78d13b30..8c988230062de22c3c4f4c1fdea12650ef66b83a